src/main/services/outputProcessors/soot/RunSootAnalysisOutputProcessor.groovy

```diff
@@ -49,8 +47,12 @@ class RunSootAnalysisOutputProcessor implements OutputProcessor {
                         try {
                             algorithmResult = algorithm.run(scenario);
                         } catch (ClassNotFoundInJarException e) {
+                            // indicates that the class doesnt exist on the generated build
                             println e.getMessage()
-                            algorithmResult = "error"
+                            algorithmResult = "not-found"
+                            if (algorithmResult instanceof NonCommutativeConflictDetectionAlgorithm) {
+                                algorithmResult = "not-found;not-found"
```

Comment:
Why are you using this double not-found?

Reply:
This ia needed because when its a NonComutativeAlgorithm, se must execute It twice, one with left -> right and other with right -> left. So when this case returns not found, it must be duplicated. I think, maybe this is not the best place to puta that logic, will try to do something better.

Reply:
@barbosamaatheus check out if this new implementation makes more sense to you

Reply:
This looks good. I didn't exist right